README.md

```diff
@@ -64,7 +69,15 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+  thingToDo();
+}
+
+function thingToDo(){
+  alert("The thing has been done.");
+}
+
+$(".hover-here-to-do-a-thing").hover(dosomething());
```

Comment:
Did you want to pass doSomething as a reference, so that it can be executed when you hover?  When you add the parens here, you are sending the result of (or whatever is returned from the) doSomething function.
I don;t see that "thingToDo" is being passed to doSomething. Please review the answer key.